CHANGELOG.md

```diff
@@ -7,16 +7,25 @@
 * [BUGFIX]
 * [BUGFIX] Add fallback for missing /proc/1/mounts #1172
 * [CHANGE] Add TCPSynRetrans to netstat default filter #1143
+* [CHANGE] Add a limit to the number of in-flight requests #1166
 * [ENHANCEMENT] Add Infiniband counters #1120
 * [FEATURE] Add a flag to disable exporter metrics #1148
 
+## 0.17.0 / 2018-11-30
+
+* [BUGFIX] collector/diskstats: don't fail if there are extra stats #1125
+* [BUGFIX] collector/hwmon\_linux: handle temperature sensor file #1123
+* [BUGFIX] collector/filesystem: add bounds check #1133
+* [BUGFIX] Fix dragonfly's CPU counting frequency #1140
+* [BUGFIX] Add fallback for missing /proc/1/mounts #1172
+
 ## 0.17.0-rc.0 / 2018-10-19
```

Comment:
We have so far removed release candidate entries from the changelogs once the main release is ready. See the Prometheus changelog. For everyone not having run the release candidates, that's a lot nicer to read and follow.

Reply:
Oh, sure, sounds fine to me.